docs/source/reproducers/01-considerations.md

```diff
@@ -14,6 +14,9 @@ The CRC VM has to be properly set up prior to running the reproducer play. You
 may get to that leveraging either the rhol_crc role from the CI Framework, or
 manually by calling `crc setup && crc start -p pull-secret`.
 
+Note: make sure to create the CRC with proper disk size it won't be resized
```

Comment:
maybe provide a reasonable disk size? IIRC we usually point to 100G ?

Reply:
done